src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -241,6 +241,11 @@ public string GetNodeLocation()
                 Trace.Info($"Found UseNode10 knob, use node10 for node tasks: {useNode10}");
                 nodeFolder = NodeHandler.node10Folder;
             }
+            if (nodeFolder == NodeHandler.nodeFolder && 
+                AgentKnobs.AgentDeprecatedNodeWarnings.GetValue(ExecutionContext).AsBoolean() == true)
+            {
+                ExecutionContext.Warning(StringUtil.Loc("DeprecatedRunner", new[] { Task.Name.ToString() }));
```

Comment:
Why `new[] { Task.Name.ToString() }`, not `Task.Name.ToString()`😃 ?

Reply:
Resolved